js/MVP/views/moteinnkallelse/components/LestInnkallelseCheckbox.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BekreftCheckboksPanel } from 'nav-frontend-skjema';
+import { useMutateBrevLest } from '../../../hooks/brev';
+import { brevTypes } from '../../../globals/constants';
+
+const texts = {
+  lestInnkallelse: 'Jeg bekrefter at jeg har lest denne innkallingen ',
+  lestEndring: 'Jeg bekrefter at jeg har lest endringen / denne meldingen',
```

Comment:
Kanskje litt kronglete tekst for lestEndring? Kanskje heller noe ala "Jeg bekrefter at jeg har lest oppdateringen av møtet", eller noe?

Reply:
Vi brukte tekster fra skisser så den var slik i Figma. Den er endret nå, både her og i `dialogmote` .

Reply:
Endret nå 👍🏻